scipy/linalg/flapack_other.pyf.src

```diff
@@ -2308,17 +2308,17 @@ subroutine <prefix>larfg(n, alpha, x, incx, tau, lx)
 end subroutine <prefix>larfg
 
 subroutine <prefix>larf(side,m,n,v,incv,tau,c,ldc,work,lwork)
-    character intent(in), check(side[0]=='L'||side[0]=='R') :: side = 'L'
+    character intent(in), check(side=='L'||side=='R') :: side = 'L'
```

Comment:
e.g. this is a simple fix, previously `character` mapped to `char*` because it was treated as `character(1)` but now it correctly needs to map to `char`.